var/spack/repos/builtin/packages/libfabric/package.py

```diff
@@ -16,7 +16,7 @@ class Libfabric(AutotoolsPackage):
 
     maintainers = ['rajachan']
 
-    version('master', branch='master')
+    version('develop', branch='main')
```

Comment:
Can we call it main instead?

Reply:
+1, but this has already been taken care of in https://github.com/spack/spack/pull/32083/